### DISCUSSION TITLE
wazuh whitelist not passed to the sensors

### DISCUSSION THREAD

**Discussion by sinnetll19** [Security-Onion-Solutions/securityonion#2735] [Unsupported Versions] [answered]
when modifying /opt/so/conf/wazuh/ossec.conf on the manager to add an IP to the wazuh whitelist to prevent an active response the changes do not get transferred down to the sensors. where can i modifty wazuh whitelist changes to apply to everything at once?

**Comment by weslambert** [comment, marked as answer]
This is not currently supported, as each node runs its own Wazuh manager, so you might expect different settings for those nodes, depending on their location and their purpose, etc.

**Reply by weslambert** [reply]
This could potentially be added in as a pillar item, in addition to other items here: https://github.com/Security-Onion-Solutions/securityonion/issues/2718
